src/pages/eunjungPark/Login/Login.js

```diff
@@ -1,8 +1,114 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Login.scss';
 
 class LoginEunjung extends Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  handleInput = event => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  loginButton = () => {
+    const { id, pw } = this.state;
+    const checkId = id.includes('@');
+    const checkPw = pw.length >= 4;
+
+    if (checkId && checkPw) {
+      alert('입력성공');
+
+      fetch('http://10.58.3.176:8000/users/signin', {
+        method: 'POST',
+        body: JSON.stringify({
+          name: '이름',
+          email: this.state.id,
+          password: this.state.pw,
+          contact_mobile: '010-0000-0000',
+          nickname: 'AAA',
+          address: 'BBB',
+        }),
+      })
+        .then(response => response.json())
+        .then(response => {
+          if (response.message === 'SUCCESS') {
+            alert('로그인성공');
+            this.goToMain();
+          } else {
+            alert('로그인다시!');
+          }
+        });
+    } else {
+      if (!checkId) {
+        alert('아이디에는 @가 포함되어야 합니다.');
+      }
+
+      if (!checkPw) {
+        alert('비밀번호는 4자리 이상이어야 합니다.');
+      }
+    }
+  };
+
+  goToMain = () => {
+    this.props.history.push('/main-eunjung');
+  };
+
   render() {
-    return <div>Eunjung</div>;
+    return (
+      <div className="body-container">
```

Comment:
- 가장 바깥 태그의 클래스 네임은 컴포넌트 이름과 동일하게 지어주세요!
```suggestion
      <div className="loginEunjung">
```
- 클래스 네임은 케밥 케이스를 사용하고 계신것 같은데, 어느 것을 사용해도 상관 없으나, 다른 팀원들과 상의하여서 프로젝트 내에서 한가지 컨벤션으로 통일해주세요!